zipline/data/bundles/alpha_vantage_api.py

```diff
@@ -0,0 +1,323 @@
+# Data-Bundle for Aplhavantage
+# 
+# Usage: set api-key with 'export ALPHAVANTAGE_API_KEY=yourkey'
+#
+# Limitations: free-accounts can have 5 calls per minute and 500 calls a day.
+#   tolerance is used to wait some small additional time, otherwise sometimes
+#   we will still hit the limit if we want to squeeze out as much calls as
+#   possible.
+#   In case you have a premium-subscription, you can tune these values by
+#   setting the env-vars AV_FREQ_SEC, AV_CALLS_PER_FREQ, and AV_TOLERANCE_SEC
+#
+# Dependencies: you will need to install these packages with pip: 
+#   alpha_vantage, ratelimit
+#
+# Adjustments: to enable a bigger precision on our backtests, i decided to
+#   go for unadjusted-prices and implemented an adjustment-writer to account
+#   for dividends and splits. However, only daily-data contains this information
+#   so it's really IMPORTANT that you never only request minute-data alone.
+
+import collections
+import os
+
+import math
+import requests
+
+import numpy as np
+import pandas as pd
+
+from alpha_vantage.timeseries import TimeSeries
+
+from datetime import date, datetime, timedelta, time as dtime
+from dateutil import tz
+from trading_calendars import TradingCalendar
+from dateutil.parser import parse as date_parse
+
+from ratelimit import limits,sleep_and_retry
+
+from zipline.data.bundles import core as bundles
+from zipline.data.bundles.universe import Universe, all_alpaca_assets, get_sp500, get_sp100, get_nasdaq100
+
+AV_FREQ_SEC = os.getenv('AV_FREQ_SEC') if os.getenv('AV_FREQ_SEC') else 60
+AV_CALLS_PER_FREQ = os.getenv('AV_CALLS_PER_FREQ') if os.getenv('AV_CALLS_PER_FREQ') else 5
+AV_TOLERANCE_SEC = os.getenv('AV_TOLERANCE_SEC') if os.getenv('AV_TOLERANCE_SEC') else 1
+
+# uncomment this line if you want a static list of symbols
+#ASSETS = ['AAPL']
+ASSETS = ['AAPL', 'TSLA', 'MSFT']
+
+#ASSETS = None
+UNIVERSE = Universe.NASDAQ100
+
+def list_assets():
+    global ASSETS, UNIVERSE
+    if not ASSETS:
+        if UNIVERSE == Universe.SP100:
+            ASSETS = get_sp100()
+        elif UNIVERSE == Universe.SP500:
+            ASSETS = get_sp500()
+        elif UNIVERSE == Universe.NASDAQ100:
+            ASSETS = get_nasdaq100()
+        else:
+            ASSETS = ['AAPL']
+
+    return sorted(list(set(ASSETS)))
+
+# purpose of this function is to encapsulate both minute- and daily-requests in one 
+# function to be able to properly do rate-limiting.
+@sleep_and_retry
+@limits(calls=AV_CALLS_PER_FREQ, period=AV_FREQ_SEC + AV_TOLERANCE_SEC)
+def av_api_wrapper(symbol, interval, slice=None):
+
+    if interval == '1m':
+        ts = TimeSeries(output_format='csv')
+        data_slice, meta_data = ts.get_intraday_extended(symbol, interval='1min', slice=slice, adjusted='false')
```

Comment:
bug: you are calling this function wrong: 
![image](https://user-images.githubusercontent.com/7924802/104706846-4c91e100-5724-11eb-9a4b-cc67e04679ca.png)

it doesn't receive `adjusted`

Reply:
waiting for the latest release of alpha_vantage-package

Reply:
let's just use your version for now